src/platform/testing/e2e/helpers.js

```diff
@@ -94,7 +100,9 @@ function uploadTestFile(client, fileData) {
 function createE2eTest(beginApplication) {
   return {
     'Begin application': client => {
+      client.openUrl(BASE_URL);
       overrideSmoothScrolling(client);
+      disableAnnouncements(client);
```

Comment:
`overrideSmoothScrolling` is called manually throughout tests because it wasn't working here. It wasn't working here because it was overriding smooth scrolling without first navigating to the domain we're testing under. Navigating first to the domain, then calling `overrideSmoothScrolling` and `disableAnnouncements` _should_ fix this.